include/clad/Differentiator/Array.h

```diff
@@ -293,17 +300,19 @@ template <typename T> class array {
   }
 
   /// Negate the array and return a new array.
-  CUDA_HOST_DEVICE array_expression<T, BinarySub, array<T>> operator-() const {
-    return array_expression<T, BinarySub, array<T>>(static_cast<T>(0), *this);
+  CUDA_HOST_DEVICE array_expression<T, BinarySub, array<T> const&>
+  operator-() const {
+    return array_expression<T, BinarySub, array<T> const&>(static_cast<T>(0),
```

Comment:
```suggestion
    return array_expression<T, BinarySub, const array<T>&>(static_cast<T>(0),
```

We spell const in clad this way. Can you update the rest of the occurrences?

Reply:
fixed and also added to .clang-format

Reply:
nice!